### PR TITLE
Introduce RediSearch v2.2.7

### PR DESCRIPTION
**Pull request by rafie**

### CODE REVIEW

**Review by swilly22 — CHANGES_REQUESTED** [review]
@rafie please see comments

File: build/docker/Makefile [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
+ROOT=../..
+MK.pyver:=3
+include $(ROOT)/deps/readies/mk/main
 
-ROOT=../../
-READIES=${ROOT}/deps/readies
-include ${READIES}/mk/docker.rules
+REPO=redisgraph
+
+REDIS_VERSION=6.2.6
```
[the remark this answers is not in the document]
Comment (rafie) [reply build/docker/Makefile:8]: @swilly22 We indeed can push it on another PR, as we'll need to CP it to all integration branches.
This is the new standard procedure for Docker images generation, and the values of the parameters there determine what you're gonna get in `redislabs/redisgraph` (once one gives the right values everything works out of the box).

File: sbin/system-setup.py [hunk 2]
```diff
@@ -70,7 +73,7 @@ def install_peg(self):
             tar xzf peg.tar.gz
             cd peg-0.1.18
             make
-            make install MANDIR=.
+            $(command -v sudo) make install MANDIR=.
```
[the remark this answers is not in the document]
Comment (rafie) [reply sbin/system-setup.py:76]: Yes. 
TL;DR this is part of a larger move into non-sudo installation process.
It obviously has no effect on macOS, where almost all setup affairs are conducted without sudo, but it actually solves a huge mess on the Linux installations, in which it is practically impossible to get the Python installations right once you're in sudo. So this (along with many other changes in the automation code) allows one to run installations from non-sudo environment (and automatically entring sudo when it is necessary).

**Review by swilly22 — APPROVED** [review]
